### PR TITLE
admin: add retention-rules-extend command

### PR DESCRIPTION
**Pull request by audrium**
closes https://github.com/reanahub/reana-server/issues/556

To test:
- create some workflows with retention rules
- `kubectl exec -i -t deployment/reana-server -c rest-api -- flask reana-admin retention-rules-extend -d 4 -w d171760a-a7db-4270-80fd-dddd526bc20c`

### CODE REVIEW

**Review by mdonadoni — COMMENTED** [review]
Could you also add an entry in `CHANGES.rst`?

File: reana_server/reana_admin/cli.py [hunk 1]
```diff
@@ -762,6 +763,39 @@ def retention_rules_apply(dry_run: bool) -> None:
             )
 
 
+@reana_admin.command()
+@click.option("--workflow", "-w", help="UUID of the workflow", required=True, type=str)
+@click.option(
+    "--days", "-d", help="Number of days to extend the rules.", required=True, type=int
+)
+def retention_rules_extend(workflow: str, days: int) -> None:
```
[the remark this answers is not in the document]
Comment (audrium) [reply reana_server/reana_admin/cli.py:771]: I see it more like a feature which was described here: https://github.com/reanahub/reana-server/issues/556#issuecomment-1279168107

[the remark this answers is not in the document]
Comment (mdonadoni) [reply reana_server/reana_admin/cli.py:771]: @tiborsimko should we ask for confirmation like we do in `retention-rules-apply` when using `--force-date`?

In any case, if we accept negative values for `days`, we also need to make sure that `retention_days` does not become negative.

[the remark this answers is not in the document]
Comment (audrium) [reply reana_server/reana_admin/cli.py:771]: Changed the code to keep 1 day as a minimum value for `retention_days` as discussed this morning

**Pull request merged**